scripts/fast-reboot

```diff
@@ -3,33 +3,46 @@
 REBOOT_USER=$(logname)
 REBOOT_TIME=$(date)
 REBOOT_CAUSE_FILE="/var/cache/sonic/reboot-cause.txt"
+REBOOT_TYPE=$(basename $0)
+
+# Check reboot type supported
+case "$REBOOT_TYPE" in
+    "fast-reboot")
```

Comment:
for boot type, can we use fastboot and warmboot?

Reply:
It may confuse people, the scripts are called *-reboot.